src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public static Ball getRandomBall() {
+        Ball ball = new Ball();
```

Comment:
I'd recommend to create all-arguments constructor for Ball class and use it instead of setters here. It will make code cleaner